app/views/books/index.html.erb

```diff
@@ -1,24 +1,31 @@
-<h1>Bookstore</h1>
 <p>
-  <%= form_tag(root_path, method: :get) do %>
-    <%= text_field_tag :search %>
-    <%= submit_tag "Search" %>
+  <%= form_tag books_path, method: :get, class: 'form-inline' do %>
```

Comment:
That is because form's, by default, do their submission via a POST request. You DO need the `method: :get` here :+1: